mmdet/datasets/coco_panoptic.py

```diff
@@ -402,23 +450,41 @@ def _pan2json(self, results, outfile_prefix):
         return pan_json_results
 
     def results2json(self, results, outfile_prefix):
-        """Dump the panoptic results to a COCO panoptic style json file.
+        """Dump the results to a COCO style json file.
+
+        There are 4 types of results: proposals, bbox predictions, mask
+        predictions, panoptic segmentation predictions, and they have
+        different data types. This method will automatically recognize
```

Comment:
How about adding details for a specific data type

Reply:
I will add it